src/bin/server.rs

```diff
@@ -103,7 +103,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 async move { Service::from_blocking(handler, addr) }
             });
 
-        let addr = ([127, 0, 0, 1], port).into();
+        let addr = ([0, 0, 0, 0], port).into();
```

Comment:
Fix for error: 

```
ERROR: Service 'frontend' failed to build : The command '/bin/sh -c yarn install' returned a non-zero code: 1frontend_1  | Error proxying to http://backend:8888
frontend_1  | connect ECONNREFUSED 172.19.0.3:8888
```

Reply:
This should be put behind an environment variable that is set when run in docker. Maybe `DEV_DOCKER`. That way the server is not exposed on all interfaces when doing local development outside of docker.

The rest looks good to me. Thanks @nilslice!

Reply:
that's a good idea. I can make this update and push later this evening or tomorrow.